app/src/main/java/org/mozilla/focus/home/pocket/PocketVideoFragment.kt

```diff
@@ -53,31 +56,38 @@ class PocketVideoFragment : Fragment() {
     }
 
     override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View {
-        fun displayFeed(layout: View, videos: PocketVideosDeferred) {
-            layout.videoFeed.gridView.adapter = PocketVideoAdapter(context!!, videos, fragmentManager!!)
-        }
-
-        val layout = inflater.inflate(R.layout.fragment_pocket_video, container, false)
+        return inflater.inflate(R.layout.fragment_pocket_video, container, false)
+    }
 
-        if (deferredVideos.isCompleted) {
-            displayFeed(layout, deferredVideos)
-        } else {
-            // TODO: #769?: display loading screen.
-
-            launch(UI) {
-                val videos = deferredVideos.await()
-                if (videos != null) {
-                    displayFeed(layout, deferredVideos)
-                } else {
-                    // TODO: #769: display error screen.
-                }
+    override fun onStart() {
+        super.onStart()
+        val adapter = PocketVideoAdapter(context!!, fragmentManager!!).also {
```

Comment:
Is there any reason this should be in `onStart` instead of `onCreate`? I feel like one-time set-up code usually goes in onCreate.

Reply:
We need access to a view here, but it could be moved to `onCreateView`.  Would you prefer that I move it there?  I don't really have any preference one way or the other.